sycl/doc/extensions/experimental/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -120,23 +113,21 @@ public:
   T* operator&() const noexcept;
 
 private:
-  T* ptr; // exposition only
-
+  T storage;
```

Comment:
We don't usually document private data members in our specifications.  I think we could just delete this.

Reply:
removed